### ISSUE TITLE
Handle multichannel output images correctly

### ISSUE THREAD

**Issue by ctrueden** [imagej/imagej-omero#31]
After running e.g. "Rotate 90 Degrees Left" on an RGB image, the resultant image is only single channel.

**Comment by ctrueden** [comment]
Very likely caused by #55.

**Comment by ctrueden** [comment]
I did some testing with the latest version of ImageJ-OMERO (post #55 bug-fix). Trying to run Edit > Invert on the Clown sample from the OMERO server side, I see this exception:
```
[ERROR] Exception
	* java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
	* 	at java.util.ArrayList.rangeCheck(ArrayList.java:638) ~[na:1.8.0_11]
	* 	at java.util.ArrayList.get(ArrayList.java:414) ~[na:1.8.0_11]
	* 	at io.scif.AbstractMetadata.get(AbstractMetadata.java:122) ~[scifio-0.23.0.jar:0.23.0]
	* 	at io.scif.services.DefaultDatasetIOService.updateDataset(DefaultDatasetIOService.java:206) ~[scifio-0.23.0.jar:0.23.0]
	* 	at io.scif.services.DefaultDatasetIOService.open(DefaultDatasetIOService.java:130) ~[scifio-0.23.0.jar:0.23.0]
	* 	at io.scif.services.DefaultDatasetIOService.open(DefaultDatasetIOService.java:109) ~[scifio-0.23.0.jar:0.23.0]
	* 	at net.imagej.omero.DefaultOMEROService.downloadImage(DefaultOMEROService.java:338) ~[imagej-omero-0.3.0.jar:0.3.0]
	* 	at net.imagej.omero.DefaultOMEROService.convert(DefaultOMEROService.java:420) ~[imagej-omero-0.3.0.jar:0.3.0]
	* 	at net.imagej.omero.DefaultOMEROService.convert(DefaultOMEROService.java:430) ~[imagej-omero-0.3.0.jar:0.3.0]
	* 	at net.imagej.omero.DefaultOMEROService.toImageJ(DefaultOMEROService.java:310) ~[imagej-omero-0.3.0.jar:0.3.0]
	* 	at net.imagej.omero.ModuleAdapter.launch(ModuleAdapter.java:126) ~[imagej-omero-0.3.0.jar:0.3.0]
	* 	at net.imagej.omero.ScriptRunner.invoke(ScriptRunner.java:87) [imagej-omero-0.3.0.jar:0.3.0]
	* 	at net.imagej.omero.ScriptRunner.invoke(ScriptRunner.java:70) [imagej-omero-0.3.0.jar:0.3.0]
	* 	at net.imagej.omero.ScriptRunner.main(ScriptRunner.java:112) [imagej-omero-0.3.0.jar:0.3.0]
	* 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_11]
	* 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_11]
	* 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_11]
	* 	at java.lang.reflect.Method.invoke(Method.java:483) ~[na:1.8.0_11]
	* 	at net.imagej.launcher.ClassLauncher.launch(ClassLauncher.java:258) [imagej-launcher-3.1.6.jar:3.1.6]
	* 	at net.imagej.launcher.ClassLauncher.run(ClassLauncher.java:184) [imagej-launcher-3.1.6.jar:3.1.6]
	* 	at net.imagej.launcher.ClassLauncher.main(ClassLauncher.java:76) [imagej-launcher-3.1.6.jar:3.1.6]
```
Which is caused by an thoroughly unpopulated `OMEROFormat.Metadata` object—e.g., not even the fields like `sizeX` are set. Remote debugging works extremely well to investigate this further.

**Comment by ctrueden** [comment]
Fixed in scifio/scifio#275, pending some further discussion with @hinerm. Will close out this issue after the final solution is merged to scifio master and released.

**Comment by ctrueden** [comment]
[scifio 0.24.0](https://github.com/scifio/scifio/releases/tag/scifio-0.24.0) released and uploaded.
